tendrl/node_agent/objects/notification_message/__init__.py

```diff
@@ -0,0 +1,23 @@
+from tendrl.commons.message import Message as CommonMessage
+from tendrl.commons import objects
+
+
+class NotificationMessage(objects.BaseObject, CommonMessage):
+    internal = True
+
+    def __init__(self, **notification_message):
+        self._defs = {}
+        CommonMessage.__init__(self, **notification_message)
+        objects.BaseObject.__init__(self)
+
+        self.value = '_tendrl/notifications/{0}'
```

Comment:
Put this in /notifications/ and not _tendrl/notifications

Reply:
ok i will change this